test/functional/ui/mouse_spec.lua

```diff
@@ -23,7 +23,12 @@ describe('Mouse input', function()
     screen:attach()
     screen:set_default_attr_ids({
       [1] = {background = hlgroup_colors.Visual},
-      [2] = {bold = true}
+      [2] = {bold = true},
+      [3] = {
+        foreground = hlgroup_colors.NonText,
+        background = hlgroup_colors.Visual,
```

Comment:
These test changes should be reverted. The change should only affect cursorline, _not_ visual-selection.